### PR TITLE
Fixes : Sartaj

### PR DESCRIPTION
**Pull request by sartaj10**
These fixes are for the following tickets
- All Dashboards - Time zone information - Sartaj
- All Dashboards - Remove reference to IRNC from description headings- Sartaj

### CODE REVIEW

File: dashboards/flow-data-for-data-archives.json [hunk 1]
```diff
@@ -382,7 +382,7 @@
...
+      "content": "<center><h1><b>Flow Data for Data Archives</b></h1></center>\n<center>This dashboard provides flow data for the top 10 sender and receiver institutions by volume, rate, and retransmits for the data archives.</center><center>All times are displayed in browser local time.</center>\n\n\n<!-- Global site tag (gtag.js) - Google Analytics -->\n<script async src=\"https://www.googletagmanager.com/gtag/js?id=UA-142763676-1\"></script>\n<script>\n  window.dataLayer = window.dataLayer || [];\n  function gtag(){dataLayer.push(arguments);}\n  gtag('js', new Date());\n\n  gtag('config', 'UA-142763676-1');\n</script>",
```
Comment (daldoyle) [inline dashboards/flow-data-for-data-archives.json]: It looks like while merging conflicts you stepped on Katrina's changes. All sender/receiver should be source/destination.

File: dashboards/flow-data-per-country.json [hunk 2]
```diff
@@ -383,7 +383,7 @@
...
+      "content": "\n<center><h1><b>Flow Data by Selected Country</b></h1></center>\n<center>Select a Country from the list above. This dashboard shows the flow data for the selected country.</center>\n<center>The tables show top 10 by both volume and rate as well as top talker organizations.  Other flow stats are also included at the bottom.</center>\n<center>The sankey graph shows the top 10 senders and receivers by volume.  Thicker lines indicate greater flows.</center><center>All times are displayed in browser local time.</center>\n<!-- Global site tag (gtag.js) - Google Analytics -->\n<script async src=\"https://www.googletagmanager.com/gtag/js?id=UA-142763676-1\"></script>\n<script>\n  window.dataLayer = window.dataLayer || [];\n  function gtag(){dataLayer.push(arguments);}\n  gtag('js', new Date());\n\n  gtag('config', 'UA-142763676-1');\n</script>",
```
Comment (daldoyle) [inline dashboards/flow-data-per-country.json]: Same as above.

File: dashboards/flow-data-per-organization.json [hunk 3]
```diff
@@ -19,7 +19,7 @@
...
..."<div style=\"float:right;\">\n<a class=\"btn navbar-button gf-timepicker-nav-btn\" href=\"/grafana/d/-l3_u8nWk/individual-flows?var-src=$Organization\" style=\"position-fixed; left: -100px; width: 300px; text-decoration: none; color: #52545C; font-weight: bold;\">See Individual Flows for $Organization</a>\n</div>\n<br clear=\"both\" />\n<div style=\"padding-left: 10%;\">\n<center><h1><b>Flow Data by Organization</b></h1></center>\n<center>Select an Organization from the list above.</center>\n<center>Please note that the dropdown will only show 1000 results at a time, so you may need to filter (start typing) to find a specific organization.</center>\n<center>This dashboard shows top flows with the specified organization as the sender or receiver in a given time period.</center><center>All times are displayed in browser local time.</center>\n</div>\n\n\n\n<br>\n<!-- Global site tag (gtag.js) - Google Analytics -->\n<script async src=\"https://www.googletagmanager.com/gtag/js?id=UA-142763676-1\"></script>\n<script>\n  window.dataLayer = window.dataLayer || [];\n  function gtag(){dataLayer.push(arguments);}\n  gtag('js', new Date());\n\n  gtag('config', 'UA-142763676-1');\n</script>",
```
Comment (daldoyle) [inline dashboards/flow-data-per-organization.json]: Same as above, also squashed some of the changes I had made.

File: dashboards/flow-data.json [hunk 4]
```diff
@@ -382,7 +382,7 @@
       "type": "netsagenavigation"
     },
     {
-      "content": "<center><h1><b>Flow Data</b></h1></center>\n<center>This dashboard provides flow data for the top 10 sources and destinations by volume and rate.</center>\n<!-- Global site tag (gtag.js) - Google Analytics -->\n<script async src=\"https://www.googletagmanager.com/gtag/js?id=UA-142763676-1\"></script>\n<script>\n  window.dataLayer = window.dataLayer || [];\n  function gtag(){dataLayer.push(arguments);}\n  gtag('js', new Date());\n\n  gtag('config', 'UA-142763676-1');\n</script>",
+      "content": "<center><h1><b>Flow Data</b></h1></center>\n<center>This dashboard provides flow data for the top 10 senders and receivers by volume and rate.</center><center>All times are displayed in browser local time.</center>\n<!-- Global site tag (gtag.js) - Google Analytics -->\n<script async src=\"https://www.googletagmanager.com/gtag/js?id=UA-142763676-1\"></script>\n<script>\n  window.dataLayer = window.dataLayer || [];\n  function gtag(){dataLayer.push(arguments);}\n  gtag('js', new Date());\n\n  gtag('config', 'UA-142763676-1');\n</script>",
```
Comment (daldoyle) [inline dashboards/flow-data.json]: Same as above.

File: dashboards/flows-by-science-discipline.json [hunk 5]
```diff
@@ -383,7 +383,7 @@
...
+      "content": "\n<center><h1><b>Flow Data by Science Discipline</b></h1></center>\n<center>Select a Discipline from the dropdown above. We recommend selecting no more than 3 at a time.</center>\n<center>This sankey graph shows the top 10 sender science organizations by volume for the selected disciplines in a specific period of time.</center>\n<center>It then shows their disciplines and the top 5 receiver science organizations for each.  Thicker lines indicate larger flows.</center><center>All times are displayed in browser local time.</center>\n<!-- Global site tag (gtag.js) - Google Analytics -->\n<script async src=\"https://www.googletagmanager.com/gtag/js?id=UA-142763676-1\"></script>\n<script>\n  window.dataLayer = window.dataLayer || [];\n  function gtag(){dataLayer.push(arguments);}\n  gtag('js', new Date());\n\n  gtag('config', 'UA-142763676-1');\n</script>",
```
Comment (daldoyle) [inline dashboards/flows-by-science-discipline.json]: Same as above.

File: dashboards/individual-flows.json [hunk 6]
```diff
@@ -382,7 +382,7 @@
...
+      "content": "\n<h1><center><b>Individual Flow Dashboard</b></center></h1>\n<center>This dashboard shows per-flow level data, given a particular sender and receiver organization. \nThe table shows the volume, rate, retransmits, retransmit %, RTT and duration for each flow. (retransmit and RTT info only available for tstat data) </center>\n<center>The heatmaps below show volume, rate and retransmits of flows for the given sender and receiver organizations over a specific period of time.</center>\n<center>The horizontal axis represents days and the vertical axis represents time of day. Darker squares show higher data transmissions at those hours.</center><center>All times are displayed in browser local time.</center>\n\n\n<!-- Global site tag (gtag.js) - Google Analytics -->\n<script async src=\"https://www.googletagmanager.com/gtag/js?id=UA-142763676-1\"></script>\n<script>\n  window.dataLayer = window.dataLayer || [];\n  function gtag(){dataLayer.push(arguments);}\n  gtag('js', new Date());\n\n  gtag('config', 'UA-142763676-1');\n</script>\n",
```
Comment (daldoyle) [inline dashboards/individual-flows.json]: Same as above.

File: dashboards/flow-data-for-data-archives.json (hunk 1 above)
Comment (sartaj10) [reply dashboards/flow-data-for-data-archives.json]: Fixed

File: dashboards/flow-data-per-country.json (hunk 2 above)
Comment (sartaj10) [reply dashboards/flow-data-per-country.json]: Fixed

File: dashboards/flow-data-per-organization.json (hunk 3 above)
Comment (sartaj10) [reply dashboards/flow-data-per-organization.json]: Fixed

File: dashboards/flow-data.json (hunk 4 above)
Comment (sartaj10) [reply dashboards/flow-data.json]: Fixed

File: dashboards/flows-by-science-discipline.json (hunk 5 above)
Comment (sartaj10) [reply dashboards/flows-by-science-discipline.json]: Fixed

File: dashboards/individual-flows.json (hunk 6 above)
Comment (sartaj10) [reply dashboards/individual-flows.json]: Fixed

**Pull request merged**